gradle.properties

```diff
@@ -1,4 +1,4 @@
-PROJECT_VERSION=1.1.0
+PROJECT_VERSION=1.2.0-SNAPSHOT
```

Comment:
So when people include dependencies, it will always be suffixed with "SNAPSHOT"?